recipes/guidata/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.7.6" %}
+
+package:
+  name: guidata
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/guidata/archive/v{{ version }}.tar.gz
+  sha256: 2ffc8e6dd27a57c6ec0e260986368beeddbd2fa5588f5200614b307b27a39e37
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - guidata-tests-py2 = cguidata.tests:run  # [py2]
+    - guidata-tests-py3 = cguidata.tests:run  # [py3]
+
+requirements:
+  build:
+    - python
+    - pip
+    - spyder
+    - pyqt
+  run:
+    - python
+    - spyder
+    - pyqt
+
+test:
+  imports:
+    - guidata
+
```

Comment:
We need to test for those entrypoints. Can you please add here:

```yaml
test:
  imports:
    - guidata
  commands:
    - guidata-tests-py2 --help  # [py2k]
    - guidata-tests-py3 --help  # [py3k]
```

Not sure if that entrypoints has a `--help` options, but maybe there is something we can test.
(Again, I'm not a guidata user.)

Reply:
Are those entry points used for anything during development or they exist to run tests of the libraries? If the former, I would rather to remove them and keep `noarch: python`.

Reply:
Yep! :+1:  to that.

BTW, if that is the case you can do the same for `PythonQwt` b/c that one also has the same `entrypoints` :smile: 

Just to be clear you would need to patch `setup.py` to remove the entrypoint and re-add the `noarch` line.

Reply:
@nicoddemus they are just for testing! 

@ocefpaf exactly! For ```PythonQwt``` is the same thing.

Ok then, going to patch and add ```noarch: python``` for both!
Thanks!

:)